cltk/corpus/gujarati/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Please remove this blank line

Reply:
@the-ethan-hunt  Please, Remove this blank line.

Reply:
@LBenzahia , I tried to make the file blank, but I am unable to do it.

Reply:
fixed